operate/resource.py

```diff
@@ -132,12 +137,11 @@ def store(self) -> None:
         self.load(self.path)  # Validate before making backup
 
         for i in reversed(range(N_BACKUPS - 1)):
-            older = path.with_name(f"{path.name}.bak.{i}")
-            newer = path.with_name(f"{path.name}.bak.{i + 1}")
+            older = path.with_name(f"{path.name}.{i}.bak")
+            newer = path.with_name(f"{path.name}.{i + 1}.bak")
```

Comment:
shouldn't we call the the one with {i} newer and the {i+1} older? Because the higher numbered backup was created before the lower number.

Reply:
it's refering to older and newer names here.

Reply:
renamed for clarity